### PR TITLE
[STARTAI-14] Update README

### PR DESCRIPTION
**Pull request by nasstis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+## Dataset
+[Student Alcohol Consumption](https://www.kaggle.com/datasets/uciml/student-alcohol-consumption?select=student-mat.csv)
+
+This dataset contains information about students that may influence their academic performance.
+
+## The purpose of the prediction 
+To determine the student's final grades based on their alcohol consumption and other characteristics (listed below) using various models and our own linear regression model
+
+## Selected features for prediction
```
Comment (ArsenOOO7) [inline README.md:11]: Add all features from https://github.com/PNUAITeam13/AlcoholConsumption/blob/main/data_attributes.txt

Comment (ArsenOOO7) [reply README.md:11]: And maybe use ```-``` instead of ```+```
- **age**: Student's age (numeric: from 15 to 22)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+7. **activities**: Extra-curricular activities (binary: yes or no)
+8. **higher**: Wants to take higher education (binary: yes or no)
+9. **internet**: Internet access at home (binary: yes or no)
+10. **romantic**: With a romantic relationship (binary: yes or no)
+11. **famrel**: Quality of family relationships (numeric: from 1 - very bad to 5 - excellent)
+12. **freetime**: Free time after school (numeric: from 1 - very low to 5 - very high)
+13. **goout**: Going out with friends (numeric: from 1 - very low to 5 - very high)
+14. **Dalc**: Workday alcohol consumption (numeric: from 1 - very low to 5 - very high)
+15. **Walc**: Weekend alcohol consumption (numeric: from 1 - very low to 5 - very high)
+16. **absences**: Number of school absences (numeric: from 0 to 93)
```
Comment (ArsenOOO7) [inline README.md:28]: And selected maybe without explanation because it would be in the list above)

**Review by JJerome-NM — APPROVED** [review]
In general, kk.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+- **internet**: Internet access at home (binary: yes or no)
+- **romantic**: With a romantic relationship (binary: yes or no)
+- **famrel**: Quality of family relationships (numeric: from 1 - very bad to 5 - excellent)
+- **freetime**: Free time after school (numeric: from 1 - very low to 5 - very high)
+- **goout**: Going out with friends (numeric: from 1 - very low to 5 - very high)
+- **Dalc**: Workday alcohol consumption (numeric: from 1 - very low to 5 - very high)
+- **Walc**: Weekend alcohol consumption (numeric: from 1 - very low to 5 - very high)
+- **absences**: Number of school absences (numeric: from 0 to 93)
+
+## All features
```
Comment (ArsenOOO7) [inline README.md:30]: Please, move  ```All features``` higher than ```Selected```
Cut description from __selected__ to __all__

Comment (JJerome-NM) [reply README.md:30]: Then I think the description of the features can be moved back to the general list

**Review by ArsenOOO7 — APPROVED** [review]

**Pull request merged**